ipatests/test_integration/test_commands.py

```diff
@@ -1522,3 +1522,51 @@ def test_client_doesnot_throw_responsenotready_error(self):
         )
         # Run the command again after cache is removed
         self.master.run_command(['ipa', 'user-show', 'ipauser1'])
+
+
+class TestIPAautomount(IntegrationTest):
```

Comment:
Wouldn't that be better off in test_nfs?

Reply:
It's not actually doing any mounting, just running commands and scraping output.

Reply:
More for clarity purposes... But yeah, test_commands is fine then.